target/linux/ipq40xx/base-files/lib/upgrade/linksys.sh

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (C) 2014-2015 OpenWrt.org
+#
+
+linksys_get_target_firmware() {
+	cur_boot_part=`/usr/sbin/fw_printenv -n boot_part`
+	target_firmware=""
+	if [ "$cur_boot_part" = "1" ]
+	then
```

Comment:
To keep the style in line with other base-files scripts, could you please do something like

```
if [ "$cur_boot_part" = "1" ]; then
    ...

elif [ "$cur_boot_part" = "2" ]; then
...
fi
```